vscode/src/debugger/activate.ts

```diff
@@ -90,7 +90,11 @@ function registerCommands(context: vscode.ExtensionContext) {
           shots: 1,
           ...config,
         },
-        options,
+        {
+          // no need to save the file, in fact better not to, since it may cause the document uri to change
+          suppressSaveBeforeStart: true,
```

Comment:
Do we need to specify this here? We already set this in the package.json (search for `"debug.saveBeforeStart": "none"`).

Reply:
Yeah, I should have explained this a little better. I'm changing how we pass `programUri` here, to make the command invocation work from the webview. But by doing this, I'm breaking the "save before start" scenario (when the document uri changes after debugging starts). We do turn off "save before start" in the settings, but in theory the user could override that setting. Here, I'm disabling it completely, even if the user tries to opt in, since it's not going to work. Unlikely scenario, but thought I'd cover all bases.